src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,87 @@
+.card {
+  width: $card-size;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    display: block;
+    margin: 0 auto;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__text {
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin: 0;
+    margin-top: 40px;
+    color: $main-text-color;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    color: $second-text-color;
+    margin-top: 4px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 16px;
+
+    &-reviews {
+      font-weight: 400;
+      font-size: 10px;
+      color: $main-text-color;
+      display: flex;
+      align-items: center;
+    }
+  }
```

Comment:
```suggestion
  &__rating {
    display: flex;
    justify-content: space-between;
    margin-top: 16px;
  }

  &__rating-reviews {
    font-weight: 400;
    font-size: 10px;
    color: $main-text-color;
    display: flex;
    align-items: center;
  }
```

Reply:
Wrong nesting usage, use it for pseudoclasses/pseudoelements, nested elements or modifiers but not for words combining. When I see this nested class I expect that it's a nested element, but it's just an element with same word in class name